### ISSUE TITLE
Add documentation about the directory

### ISSUE THREAD

**Issue by rnestler** [SpaceApi/directory#166]
Currently I can't find documentation about the directory anywhere, not even on how to retrieve the directory. I'm I just blind or is it really not documented anywhere?

**Comment by dbrgn** [comment]
I'm a bit confused, you opened an issue on the repository that _contains_ the directory.

On the website, it's linked here: https://spaceapi.io/write-software/ We could add a more prominent link on the home page (where we explain what the directory is). Should we move this issue to the website repository?

(Of course we could also add a link to https://directory.spaceapi.io/ to the README in this repo, a PR would probably have been faster than an issue 😄)

**Comment by rnestler** [comment]
> I'm a bit confused, you opened an issue on the repository that contains the directory.
>
> On the website, it's linked here: https://spaceapi.io/write-software/ We could add a more prominent link on the home page (where we explain what the directory is). Should we move this issue to the website repository?

Yes I should probably have opened it on the website. In general I find it hard to find the relevant information since we redesigned the website.
